mathics/builtin/base.py

```diff
@@ -677,6 +677,15 @@ class BoxConstruct(InstanceableBuiltin):
 
     def __new__(cls, *elements, **kwargs):
         instance = super().__new__(cls, *elements, **kwargs)
+        instance.summary_text = "box construct"
```

Comment:
It might be nice to add for _which_ box  box this is for, e.g. "box construct for Arrow"

Reply:
@rocky ``box representation for {a/an} {name of the class without Box}`` seems to work, isn´t it?

Reply:
One improvement would be to split names like ``CompiledCode`` as  ``compiled code``, but this could be a second round.

Reply:
> @rocky `box representation for {a/an} {name of the class without Box}` seems to work, isn´t it?

Yes, that would be awesome.

Reply:
> > @rocky `box representation for {a/an} {name of the class without Box}` seems to work, isn´t it?
> 
> Yes, that would be awesome.
Done.